src/views/Space/Overview.vue

```diff
@@ -20,6 +22,7 @@ onMounted(() => {
 const spaceIdComposite = `${props.space.network}:${props.space.id}`;
 
 const spaceStarred = computed(() => spacesStore.starredSpacesIds.includes(spaceIdComposite));
+const spaceIsEditable = computed(() => props.space.controller === web3.value.account);
```

Comment:
```suggestion
const spaceIsEditable = computed(
  () => props.space.controller.toLowerCase() === web3.value.account.toLowerCase()
);
```

For Ethereum we might need to convert it to lowercase. I think that overall we should have some kind of helper for comparing addresses (that ignores 0-pads, casing etc.), normalizing it on server-side is okay, but I imagine there may be more bugs like this because some new source uses different format (let's say if ArgentX starts padding addresses).
What do you think @bonustrack?

Reply:
Yes I think it makes sense to use lowercase to check if you are owner on the UI, some wallet may expose a lowercased address. I would just enforce the right casing / padding on the API.